로또-최고순위-최저순위.js

```diff
@@ -0,0 +1,28 @@
+function solution(lottos, win_nums) {
+    
+    const answer = [];
+    let num = 0;
+    let countzero = 0;
+    let countmin = 0;
```

Comment:
1. 불필요한 줄바꿈 하지않기
2. 배열은 const로 선언해 주기
3. 변수 선언시 Camel case로 선언해 주기 (모른다면 Camel case 공부할 것!)

```suggestion
    const answer = [];
    let num = 0;
    let countZero = 0;
    let countMin = 0;
```